lib/workers/repository/model/commit-message.ts

```diff
@@ -0,0 +1,39 @@
+export class CommitMessage {
```

Comment:
I don't see much value for this class. Why not build the final message on build command of builder?

Reply:
Yeah, I agree. I have merged functionality of both classes in `CommitMessage`